bootstrap/bootstrap.py

```diff
@@ -90,7 +93,47 @@ def validate_host():
             print("For local development, see http://tljh.jupyter.org/en/latest/contributing/dev-setup.html")
         sys.exit(1)
 
+class LoaderPageRequestHandler(SimpleHTTPRequestHandler):
+    def do_GET(self):
+        if self.path == "/logs":
+            with open("/opt/tljh/installer.log", "rb") as log_file:
+                content = log_file.read()
+                self.wfile.write(content)
+        elif self.path == "/index.html" or self.path == "/favicon.ico":
+            return SimpleHTTPRequestHandler.do_GET(self)
+        elif self.path == "/":
+            self.send_response(302)
+            self.send_header('Location','/index.html')
+            self.end_headers()
+        else:
+            SimpleHTTPRequestHandler.send_error(self, code=403)
+
+def serve_forever(server):
+    try:
+        server.serve_forever()
+    except KeyboardInterrupt:
+        pass
+
 def main():
+    temp_page_flag = "--temporary-page"
+    if temp_page_flag in sys.argv:
```

Comment:
Since `argparse` is in the standard library, what do you think of using that here? Would be cleaner than directly looking in `sys.argv`. However, we do pass all params through to installer, so if it is difficult we should add a comment here on why this isn't using argparse, and let it go :)

Reply:
I've added a comment about not using `argparse` :sweat_smile:

The reasoning for that was that if using `argparse` here for the temporary page flag, all the other flags should be defined here also (I got complaints about not knowing the other args).  And I think apart from this temp page all of the flags are installer-specific. WDYT?